models/report.go

```diff
@@ -147,7 +139,14 @@ func (r Report) date() time.Time {
 	return t
 }
 
-func WithDerivedAttributes(r Report) Report {
+func (r Report) Performance() float64 {
+	if r.EventIndexRate != nil {
+		return *r.EventIndexRate
+	}
+	return 0
```

Comment:
Could `EventIndexRate` be a `float64` instead of a `*float64`?

Reply:
`EventIndexRate` is calculated from a division. If the denominator is zero, then this is nil (which is semantically different from being 0), so its needs a pointer